configs/pie-components-config/vite.config.js

```diff
@@ -22,7 +22,7 @@ const sharedConfig = ({ build = {}, plugins = [], ...rest }) => defineConfig({
         },
         rollupOptions: {
             external: (id) => {
```

Comment:
Having a comment explaining why we use those `external` configs can be quite handy in the future.

Reply:
good idea! is it okay if i address that in a follow up PR?

Reply:
For sure!